test_solution.sh

```diff
@@ -101,6 +101,15 @@ case "${EXTENSION}" in
             COMPILER="rustc"
             COMPILER_FLAGS=(-O -C target-cpu=native)
         fi
+        if [ "${EXTENSION}" = "hs" ]; then
+            LANGUAGE="Haskell"
+            COMPILER="ghc"
+        fi
```

Comment:
I think this is not needed as it's duplicated.

Reply:
Thanks, I messed up my branches